borg/crypto.pyx

```diff
@@ -124,7 +143,7 @@ cdef class AES:
         if not out:
             raise MemoryError
         try:
-            if not EVP_DecryptUpdate(&self.ctx, out, &outl, data, inl):
+            if not EVP_DecryptUpdate(&self.ctx, out, &outl, <const unsigned char*> data_buf.buf, inl):
```

Comment:
Here's the C code Cython generates for this:

```
__pyx_t_3 = ((!(EVP_DecryptUpdate((&__pyx_v_self->ctx), __pyx_v_out, (&__pyx_v_outl), ((unsigned char const *)__pyx_v_data_buf.buf), __pyx_v_inl) != 0)) != 0);
```

It's really a straight C-cast of data_buf.buf